src/Widgets/ToolbarButton.vala

```diff
@@ -80,12 +80,14 @@ public class ENotes.ToolbarButton : Gtk.Button {
 
             if (code_buffer.has_selection) {
                 var text = start.get_text (end);
-
                 if (this.type == 2) {
                     plugin.request_string (text);
                 } else {
+                    WordWrapper wrapper = new WordWrapper (ext);
```

Comment:
This won't apply after a comment above, but in Vala you can use `var`, specially if the type is right after it :)